git/keys.go

```diff
@@ -8,7 +8,7 @@ import (
 )
 
 // keySize is the size of generated private keys.
-var keySize = 4096
+var keySize = 2048
```

Comment:
a) that is OK if 4096 it's what github recommends
b) fine; set it for the tests only
c) set it for the tests only